### PR TITLE
Implement protobuf stub generation through protoc-tools

### PR DESCRIPTION
**Pull request by madhukaw**

### CODE REVIEW

File: native/src/main/java/io/ballerina/protoc/builder/syntaxtree/utils/ClientUtils.java [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+                getFieldAccessExpressionNode("self", "sClient"),
+                getSimpleNameReferenceNode("sClient")
+        );
+        function.addQualifiers(new String[]{"isolated"});
+        return function;
+    }
+
+    private static Function getSendFunction(Method method) {
+        String inputCap;
+        switch (method.getInputType()) {
```
Comment (kalaiyarasiganeshalingam) [inline native/src/main/java/io/ballerina/protoc/builder/syntaxtree/utils/ClientUtils.java]: Can't we move to another method to remove [duplicates](https://github.com/ballerina-platform/protoc-tools/pull/1/files#diff-f335938c6a0051a6339c6d27b866c0af2dfb229e530dded64093cd5bdef8416bR189)? 

Check other places as well

File: .gitignore [hunk 2]
```diff
@@ -21,3 +21,32 @@
...
+*.DS_Store
+
+# Ballerina
+velocity.log*
+*Ballerina.lock
+
+# Auto generated Dependencies.toml files
+examples/**/Dependencies.toml
+load-tests/**/Dependencies.toml
+integration-tests/Dependencies.toml
```
Comment (dilanSachi) [inline .gitignore:51]: we can remove these right?

File: gradle.properties [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+#dependency versions
+ballerinaLangVersion=2201.2.0-20220607-123700-52fedf4d
+ballerinaTomlParserVersion=1.2.2
+checkstylePluginVersion=8.18
+commonsLang3Version=3.8.1
+slf4jVersion=1.7.30
+protoGoogleCommonsVersion=1.17.0
+protobufJavaVersion=3.20.1
+jknackHandlebarsVersion=4.0.6
+nettyVersion=4.1.77.Final
```
Comment (dilanSachi) [inline gradle.properties:13]: do we need netty as a dependency?

File: native/src/test/resources/testng.xml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  ~
+  -->
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="ballerina-test-suite">
+    <listeners>
+        <listener class-name="io.ballerina.protoc.util.TestNGListener" />
+    </listeners>
+    <test name="grpc-native-test-suite" preserve-order="true" parallel="false">
```
Comment (dilanSachi) [inline native/src/test/resources/testng.xml]:
```suggestion
    <test name="protoc-tools-test-suite" preserve-order="true" parallel="false">
```

**Pull request closed without merging**